src/oss-find-squats-lib/Mutators/SeparatorMutator.cs

```diff
@@ -12,7 +12,10 @@ namespace Microsoft.CST.OpenSource.FindSquats.Mutators
     /// </summary>
     public class SeparatorMutator : IMutator
     {
-        public MutatorType Kind { get; } = MutatorType.Separator;
+        /// <summary>
+        /// Gets the MutatorType as <see cref="MutatorType.SeparatorRemoved"/>. It can however also be <see cref="MutatorType.SeparatorChanged"/>.
```

Comment:
What does this comment mean?

Reply:
So previously we considered both Removing a separator, and Changing a separator as a Separator Mutator. As `Kind` only accepts one for the class, I had picked SeparatorRemoved, but this doesn't make a lot of sense, so I will go split it into two mutators.